homeassistant/components/light/knx.py

```diff
@@ -109,15 +111,21 @@ def should_poll(self):
     @property
     def brightness(self):
         """Return the brightness of this light between 0..255."""
-        return self.device.current_brightness \
-            if self.device.supports_brightness else \
-            None
+        if self.device.supports_color:
+            return max(self.device.current_color) \
+                if self.device.current_color is not None else \
+                DEFAULT_BRIGHTNESS
```

Comment:
Why return a default brightness? just return `None` and Home Assistant will not set it?

Reply:
That's required internally in async_turn_on, in case there is a command to set the HS color without providing the brightness at the same time. I could return None here and use a default brightness in async_turn_on instead if this is an issue for Home Assistant.